controller/handler/endpoint_status_test.go

```diff
@@ -153,7 +158,13 @@ func TestEndpointStatuses(t *testing.T) {
 	// Can't be bothered dealing with timezone issues on the worker that runs the automated tests
 	firstResult.Timestamp = time.Time{}
 	secondResult.Timestamp = time.Time{}
-	router := CreateRouter(&config.Config{Metrics: true})
+	router := CreateRouter(&config.Config{
+		Metrics: true,
+		Storage: &storage.Config{
+			MaximumNumberOfResults: storage.DefaultMaximumNumberOfResults,
+			MaximumNumberOfEvents:  storage.DefaultMaximumNumberOfEvents,
+		},
```

Comment:
This should not be necessary. 

For test cases, the default values should be set in `storage`'s `Initialize` function, which is called by `https://github.com/TwiN/gatus/blob/3350e8144376c1fbe002469248af0427cfe985fa/storage/store/store.go#L75-L85 for tests.

As for how it should be handled for non-test cases: https://github.com/TwiN/gatus/blob/3350e8144376c1fbe002469248af0427cfe985fa/storage/config.go#L31 (the latter seems to have already been done)

Reply:
After reviewing changes further down, it seems like you've also done the change in `storage.Initialize`, so I'm not sure defining a storage config here is even necessary 🤔

Reply:
So, I cannot just rely on `Initialize` function because I need the config initialised before `store.Get()` call here https://github.com/TwiN/gatus/pull/476/files#diff-8cfab31fa8754c8f5da1e4f0222ea131e70dead2ebfb3a77b14950919627026dR103